middlewares/validators/user.js

```diff
@@ -1,6 +1,7 @@
 const joi = require('joi')
 
 const updateUser = async (req, res, next) => {
+  const userStatusEnum = ['ooo', 'idle', 'active']
```

Comment:
can we please have these under `constants` folder inside a new file called `users.js` maybe?

Reply:
Created a new file as " users.js" for constant values of status.

https://github.com/Real-Dev-Squad/website-backend/pull/348/commits/1f0c6c2a1102c0558dfc2d1cf8cfcc4d9a8ab8d5